src/couch_expiring_cache/test/couch_expiring_cache_tests.erl

```diff
@@ -0,0 +1,85 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_expiring_cache_tests).
+
+
+-include_lib("couch/include/couch_db.hrl").
+-include_lib("couch/include/couch_eunit.hrl").
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
Any chance you could make this one of the new shiny exunit tests instead?

Reply:
I would love nothing more than to use elixir here, but the expiring cache has no HTTP interface, so unfortunately it would be extremely inconvenient :(

Reply:
Its possible to do that now https://github.com/apache/couchdb/tree/prototype/fdb-layer/test/elixir#how-to-write-exunit-tests